backend/app/schemas.py

```
@@ -106,53 +122,51 @@
-    rating: Optional[float]
     approved: bool
 
-
-class UserUpdate(BaseModel):
-    full_name: Optional[str] = None
-    address: Optional[str] = None
-    phone: Optional[str] = None
-
-
-class OrderItemCreate(BaseModel):
+class ProductUpdate(ProductCreate):
```

Ruff (0.8.2)

Redefinition of unused `ProductUpdate` from line 125

(F811)